### DISCUSSION TITLE
Show size of actual directory

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#748] [1. Q&A]
Could someone tell me lua code, how to display the size of the directory currently at the cursor by pressing space (as in the case of sort by size, but only for the hovered directory). So space will get hovered directory size and make select file/directory too).

**Comment by SmetDenis** [comment]
+1

**Comment by sxyazi** [comment]
Supported in https://github.com/sxyazi/yazi/pull/2695
